### PR TITLE
Core 6943 Add preference to disable email notifications for URL uploads

### PR DESCRIPTION
**Pull request by sri2k1us**
Includes UI and service changes for this issue.

### CODE REVIEW

File: ui/de-lib/src/main/java/org/iplantc/de/client/models/UserSettings.java [hunk 1]
```diff
@@ -83,11 +85,17 @@ public void setValues(Splittable split) {
...
+            setEnableAnalysisEmailNotification(split.get(EMAIL_ANALYSIS_NOTIFCATOIN).asBoolean());
         } else {
-            setEnableEmailNotification(true);
+            setEnableAnalysisEmailNotification(true);
         }
+        if (split.get(EMAIL_IMPORT_NOTIFICATION) != null) {
+            setEnableAnalysisEmailNotification(split.get(EMAIL_IMPORT_NOTIFICATION).asBoolean());
+        } else {
+            setEnableAnalysisEmailNotification(true);
+        }
```
Comment (psarando) [inline ui/de-lib/src/main/java/org/iplantc/de/client/models/UserSettings.java]: Looks like these should be `setEnableImportEmailNotification` calls.

Comment (sri2k1us) [inline ui/de-lib/src/main/java/org/iplantc/de/client/models/UserSettings.java]: Lol. Yes!

File: ui/de-lib/src/main/java/org/iplantc/de/theme/base/client/desktop/DesktopContextualHelpMessages.properties [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
+notifyEmailHelp = <p>This option will send you an email when your analysis or URL import is complete or has failed. It will be sent to the email address you used to register for your CyVerse account.</p></br><p>This option is helpful if you would like to track the status of your analysis or URL import while outside of the Discovery Environment.</p>
 rememberFileSelectorPathHelp = <p>This option allows the Data Manager to automatically navigate the file tree to the file location that was used the last time you ran an app.</p>   <p> This option is helpful if you have an extensive file tree or often use the same input file location.</p>
 saveSessionHelp = <p>Enabling the Save session option will restore your Discovery Environment desktop to exactly how you left it when you last logged out.</p>
 defaultOutputFolderHelp = <p>This will be the default location where all outputs from your analyses can be found. </p> <p> You can keep the default path or click Browse to select a new location.</p>
+notifyEmail = Email me when analysis / URL import status changes.
```
Comment (psarando) [inline ui/de-lib/src/main/java/org/iplantc/de/theme/base/client/desktop/DesktopContextualHelpMessages.properties]: Minor, but no newline at the end of the file.

**Comment by psarando** [comment]
I didn't catch any more obvious issues :shipit:

**Comment by psarando** [comment]
Closing to open a new PR against `dev`.

**Pull request closed without merging**